packages/scripts/src/commands/app-data/convert/processors/flowParser/parsers/default.parser.spec.ts

```diff
@@ -1,3 +1,135 @@
-describe("default Parser", () => {
-  it("TODO - add tests", () => expect(true).toEqual(true));
+import { FlowTypes } from "data-models";
+import { DefaultParser } from "./default.parser";
+
+const testInputSources = {
+  data_list: { test_data_list: [{ id: 1 }, { id: 2 }, { id: 3 }] },
+};
+
+describe("Default Parser", () => {
+  it("Cleans field values - removes whitespace within empty strings", () => {
+    const parser = new DefaultParser({ processedFlowHashmap: testInputSources } as any);
+    const output = parser.run({
+      flow_name: "test_flow_input_empty",
+      flow_subtype: "generator",
+      flow_type: "generated",
```

Comment:
This isn't a valid flow type. I would recommend to use "data_list" here (as it is the most permissive type). You can also get rid of the `flow_subtype` field entirely as it isn't requried